### ISSUE TITLE
Add GitHub integration to Sentry

### ISSUE THREAD

**Issue by kfrz** [department-of-veterans-affairs/va.gov-team#454]
## User Story
As a VSP Triage team member, I can utilize the features of Sentry's GitHub integration so that assigning issues to specific owners is streamlined.

## Goal
GitHub and Sentry are linked and communicating to each other

## Acceptance Criteria
- [ ] Integrated issue tracking is enabled and configured
- [ ] Integrated release tracking is enabled and configured
- [ ] Integrated repository tracking is enabled and configured

## Definition of Done
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by omgitsbillryan** [comment]
### Current Status
TL;DR - setup of the Github-to-Sentry integration appears to have succeeded (via the UI and lack of error logs)... except that Sentry still seems to think that no integration exists

- [A github app was created](https://github.com/department-of-veterans-affairs/va.gov-team/issues/302#issuecomment-510600778) 
- An endpoint was added into the dev revproxy to accept POST requests from our GH app
- [Sentry is successfully receiving webhook events](https://github.com/organizations/department-of-veterans-affairs/settings/apps/vfssentrywebhook/advanced)
- The Sentry flow of installing the GH integration succeeded in the UI (requires a `department-of-veterans-affairs-admin` see @lisac-usds )
  - No errors found in error logs

I'm basically stumped at this point, but I'm not the only one having trouble setting up this integration for an on-premise installation of Sentry. I think it may be wise to let this feature become more mature before we try setting up this integration again.

**Comment by kfrz** [comment]
I vote we put it in the icebox for now, keep an eye out for the next version of Sentry on-premise and revisit at that point .

For now we can connect issues at a process level, including a link to the GitHub issue in the Sentry comments, and vice-versa.
